docs/source/upgrade-guide/index.md

```diff
@@ -28,6 +28,18 @@ Thus it is safe to run it on top of your project and answer the prompts.
 
 ## Upgrading to Volto 18.x.x
 
+### Node.js version support: adding 22, dropping 18
+
+Added support for Node.js 22, since it will become LTS on 2024-10-29.
+
+Long Term Support (LTS) for {term}`Node.js` 18 by the Node.js community will end on 2025-04-30.
```

Comment:
```suggestion
Long-term support for {term}`Node.js` 18 by the Node.js community will end on 2025-04-30.
```

Reply:
This is a branding thing by Node.js, so "Long Term Support (LTS)" in this context is correct.

Reply:
@stevepiercy Is it? https://nodejs.org/en/about/previous-releases says "long-term support" just like I did.

Reply:
They use both: https://github.com/nodejs/release?tab=readme-ov-file#release-phases

Reply:
Then I'd like to pick the one that seems more correct grammatically, which is "long-term support." https://www.grammarly.com/blog/punctuation-capitalization/hyphen-with-compound-modifiers/

Reply:
Actually, it depends on context. I read it again, and you're correct, because it's a kind of support being offered, not a branded version being offered.